lib/article_json/export/common/html/elements/image.rb

```diff
@@ -18,13 +26,18 @@ def export
               end
             end
 
-            private
```

Comment:
Was there any reason to make all methods public? If not, I like the idea of keeping the public API of our classes as small as possible and therefore keeping all these methods `private`.

Reply:
No, this was a mistake, changing it...

Reply:
Sorry I got confused by your comment, but the private keyword is not removed, it's added before:
https://github.com/Devex/article_json/pull/139/files#diff-5f912eaf379ce4c0a5c00289a1442812R17

Reply:
Ouch, right – I must have been blind 🙈 Sorry!